content/doc/tutorials/_download-and-run-jenkins-in-docker.adoc

```diff
@@ -112,21 +112,20 @@ which you ran the `docker run ...` command above.
 To restart the Jenkins/Blue Ocean Docker container, run the same `docker run
 ...` command you ran in step 4 (above).
 
-*Note:* You can also install Jenkins locally by:
 
-. Downloading the http://mirrors.jenkins.io/war-stable/latest/jenkins.war[latest
-  stable Jenkins WAR file] to a directory you wish to run Jenkins from - for e.g.
-* On macOS - `/Users/<your-username>`
-* On Linux - `/home/<your-username>`
-* On Windows - `C:\Users\<your-username>` +
-where `<your-username>` is your user account's name on your operating system.
+[NOTE]
+====
+If you don't wish to use Docker, you can also run Jenkins locally by:
```

Comment:
Thanks for this - I like the way you've given more purpose to this note than what I had before.

However, since this tutorial still requires Docker (because running the Pipeline further down results in Jenkins downloading the Maven Docker image and running it as a container), I'm rewording this slightly. I'm also mentioning the Java 8 _is required_ when running Jenkins this way.